src/main/java/com/internet/shop/controllers/admin/GetCurrentUserOrdersController.java

```diff
@@ -27,6 +27,6 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         List<Order> orders = orderService.getUserOrders(userId);
         req.setAttribute("userName", userService.get(userId).getName());
         req.setAttribute("orders", orders);
```

Comment:
you should take user id from session by now

Reply:
okey, you are doing this already in GetLoggedUserOrdersController. So let's rename this one, cause now it has confusing name

Reply:
Renamed to GetUserOrdersController
Is it correct?